src/App.js

```diff
@@ -65,6 +67,7 @@ function App() {
   };
 
   useEffect(() => {
+    video.current.playbackRate = defaultPlaybackRate;
```

Comment:
how do we know video is not null here?
should proabably move this inside the conditional block